### PR TITLE
Add Brakedown multilinear PCS

### PR DESCRIPTION
**Pull request by autquis**
So, this is [Brakedown](https://eprint.iacr.org/2021/1043)!

@mmagician @Antonio95

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (master)
- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Wrote unit tests
- [x] Updated relevant documentation in the code
- [ ] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

**Review by Antonio95 — APPROVED** [review]

File: poly-commit/src/linear_codes/multilinear_brakedown/mod.rs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+use ark_ff::{Field, PrimeField};
+use ark_poly::{MultilinearExtension, Polynomial};
+use ark_std::log2;
+use ark_std::marker::PhantomData;
+use ark_std::rand::RngCore;
+use ark_std::vec::Vec;
+
+mod tests;
+
+/// The univariate Brakedown polynomial commitment scheme based on [[Brakedown]][bd].
```
Comment (mmagician) [inline poly-commit/src/linear_codes/multilinear_brakedown/mod.rs:16]:
```suggestion
/// The multilinear Brakedown polynomial commitment scheme based on [[Brakedown]][bd].
```

**Comment by Pratyush** [comment]
What is the status of this PR? Would be a good idea to get it merged and then iterate on it.

**Review by Pratyush — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -183,7 +183,12 @@ Unless you explicitly state otherwise, any contribution that you submit to this
 [aurora-light]: https://ia.cr/2019/601
 [pcd-acc]: https://ia.cr/2020/499
 [pst]: https://ia.cr/2011/587
+<<<<<<< HEAD
 [brakedown]: https://ia.cr/2021/1043
+=======
+[ligero]: https://ia.cr/2022/1608
+[hyrax]: https://eprint.iacr.org/2017/1132
+>>>>>>> ligero-uni-and-ml
```
Comment (Pratyush) [inline README.md:191]:
```suggestion
[brakedown]: https://ia.cr/2021/1043
[ligero]: https://ia.cr/2022/1608
[hyrax]: https://eprint.iacr.org/2017/1132
```

Comment (autquis) [reply README.md:191]: Ah, thanks!

**Comment by autquis** [comment]
The auto-merge got disabled, so I have to mention here that the PR is ready to be merged :)

**Pull request merged**